mobile.js

```diff
@@ -116,4 +116,24 @@ submit.addEventListener('click', (event) => {
     return false;
   }
   return true;
+});
+
+const userInput = document.querySelectorAll('.dataEnter');
+
+let data = {
+};
+
+/// /// local storage \\\\\\
```

Comment:
[OPTIONAL] Remove the commented stuff ❌ as it can make the code more difficult to read, and this comment is not really necessary here. 😉